libs/features/mas/web-components/src/merch-card.js

```diff
@@ -90,6 +90,7 @@ export class MerchCard extends LitElement {
             reflect: true,
         },
         merchOffer: { type: Object },
+        analyticsId: { type: String, attribute: 'daa-lh', reflect: true },
```

Comment:
you could reuse constant from hydrate for daa-lh

Reply:
I really think that we should not use. `daa-*`  to avoid conflict with martech, since here it is merely a metadata to be consumed by CCD consumer.

Reply:
the plan is that it will be not just metadata, but also a field for martech (so the conflict is intended) 
e.g. for plans these attributes will serve the analytics values for martech